libs/helm/sidebar/src/lib/hlm-sidebar.ts

```diff
@@ -34,7 +35,8 @@ import type { ClassValue } from 'clsx';
 					data-slot="sidebar"
 					data-sidebar="sidebar"
 					data-mobile="true"
-					class="bg-sidebar text-sidebar-foreground h-screen w-[var(--sidebar-width-mobile)] p-0 [&>button]:hidden"
+					class="bg-sidebar text-sidebar-foreground h-screen w-[var(--sidebar-width)] p-0 [&>button]:hidden"
```

Comment:
I replaced `w-[var(--sidebar-width-mobile)]` with `w-[var(--sidebar-width)]`, looks like shadcn/ui only uses `SIDEBAR_WIDTH_MOBILE` to override the `--sidebar-width` with it

https://github.com/shadcn-ui/ui/blob/fc27ba269217ade8ae3a87724259874da799cd03/apps/v4/registry/new-york-v4/ui/sidebar.tsx#L190-L195